test/test_cmake/main.cpp

```diff
@@ -0,0 +1,5 @@
+#include <boost/integer.hpp>
+
+int main() {
+
```

Comment:
Without adding code that the Boost.Integer link library depends on, this may not be a complete test.

Reply:
Nevermind, I see there's no link library. :)